index.html

```diff
@@ -82,3 +82,4 @@
     </div>
 </body>
 </html>
+            <li><a href="https://github.com/bannu6301418535/hacktoberfest2019-participants.git">bannu</a></li>
```

Comment:
this line should be inside `<ul>` & `</ul>` tags.
```suggestion
            <li><a href="https://github.com/bannu6301418535">bannu</a></li>
```